Source/Mosa.BareMetal.CoolWorld/Display.cs

```diff
@@ -2,6 +2,9 @@
 
 using System.Drawing;
 using Mosa.DeviceSystem;
+using Mosa.DeviceSystem.Devices.Graphics;
+using Mosa.DeviceSystem.Drivers.Graphics;
```

Comment:
Duplicate

Reply:
one is Devices.Graphics while the other is Drivers.Graphics